pkg/app/app.go

```diff
@@ -0,0 +1,42 @@
+package app
+
```

Comment:
We may merge this new app package and examples package together later If they are closely related.

Reply:
Yes, that's the plan. Once all the tests are refactored, we can remove the examples, and just keep the app package.